app/src/main/java/com/safetyheads/akademiaandroida/fragments/technologystack/TechnologyStackViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.safetyheads.akademiaandroida.fragments.technologystack
+
+import androidx.lifecycle.ViewModel
+import com.safetyheads.akademiaandroida.dropdownlist.dropdown.ParentModel
+import kotlinx.coroutines.flow.*
+
+class TechnologyStackViewModel(private val loadItemsUseCase: TechnologyStackUseCase) : ViewModel() {
+
+    private val selectedTab = MutableStateFlow(TechnologyStackTab.Mobile)
```

Comment:
this seems like it doesn't need to be a MutableStateFlow, this could be a simple variable

Reply:
I used MutableStateFlow so that the items in the list are updated automatically whenever selectedTab has changed. If you want i can move this code to tabSelected method.